### PR TITLE
Added authSource configuration to the "Start from Scratch" section for MongoDB.

### PR DESCRIPTION
**Pull request by yash-278**
### Update `100-connect-your-database-typescript-mongodb.mdx`

Added `authSource` configuration to the "Start from Scratch" section for MongoDB.

**Reason for this update:** The `authSource` option was not mentioned in the "Start from Scratch" section, which could be particularly crucial for new developers. Including this tip here can help address potential authentication issues for those setting up their environment from the beginning, whereas experienced developers might already be familiar with such details.

Reference: [Issue](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283)

### CODE REVIEW

**Comment by yash-278** [comment]
@Dev-Connections Kindly check if this is a valid change.

**Comment by yash-278** [comment]
Not sure what the convention is to bring attention from Dev team for PR reviews, so hence I'm tagging 😅

 @jharrell

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx [hunk 1]
```diff
@@ -39,3 +39,5 @@ Here's a short explanation of each component:
 - `HOST`: The host where a [`mongod`](https://docs.mongodb.com/manual/reference/program/mongod/#mongodb-binary-bin.mongod) (or [`mongos`](https://docs.mongodb.com/manual/reference/program/mongos/#mongodb-binary-bin.mongos)) instance is running
 - `PORT`: The port where your database server is running (typically `27017` for MongoDB)
 - `DATABASE`: The name of the database
+
+> If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by [adding](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) `?authSource=admin` to the end of the connection string.
```
Comment (jharrell) [inline content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx]:
```suggestion
> If you see the following error: `Error in connector: SCRAM failure: Authentication failed.`, you can specify the source database for the authentication by adding `?authSource=admin` to the end of the connection string. See [this GitHub discussion](https://github.com/prisma/prisma/discussions/9994#discussioncomment-1562283) for additional details.
```

**Review by jharrell — CHANGES_REQUESTED** [review]
Thank you for your contribution! Made one suggestion.

File: content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx (hunk 1 above)
Comment (yash-278) [reply content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database-typescript-mongodb.mdx]: Added, Thanks!

**Review by jharrell — APPROVED** [review]

**Pull request merged**